Calibration/EcalAlCaRecoProducers/python/ALCARECOEcalCalIsolElectron_Output_cff.py

```diff
@@ -2,18 +2,6 @@
 
 # output block for alcastream Electron
 OutALCARECOEcalCalElectron_specific = cms.untracked.vstring(
```

Comment:
... forgot to remove...

--
                          Shervin
Da: Marco Musich
Inviato: lunedì 28 settembre 2015 14:53
A: cms-sw/cmssw
Rispondi a: cms-sw/cmssw
Cc: Shervin Nourbakhsh
Oggetto: Re: [cmssw] Updated ALCARECO event content: more esplicit list of collections, ma… (#11522)



In Calibration/EcalAlCaRecoProducers/python/ALCARECOEcalCalIsolElectron_Output_cff.py<https://github.com/cms-sw/cmssw/pull/11522#discussion_r40547692>:

> @@ -2,18 +2,6 @@
>
>  # output block for alcastream Electron
>  OutALCARECOEcalCalElectron_specific = cms.untracked.vstring(


what is the purpose of this now that is empty?

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/11522/files#r40547692>.